app/handler/subjects.go

```diff
@@ -95,14 +95,14 @@ func DeleteSubject(cosmos gremcos.Cosmos, w http.ResponseWriter, r *http.Request
 }
 
 func CreateSubject(cosmos gremcos.Cosmos, w http.ResponseWriter, r *http.Request) {
-	subject := model.Subject{}
+	subject := modelAPI.Subject{}
 	var err error
 	if err = json.NewDecoder(r.Body).Decode(&subject); err != nil {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
-	subject, err = dbOperations.CreateSubject(cosmos, subject)
+	res, err := dbOperations.CreateSubject(cosmos, subject)
 
 	if err != nil {
 		fmt.Println(err)
```

Comment:
for better logging (as other non-error messages might be mixed with error ones), mind using logging packages like logrus. This way you can log debug, error, warn distinctively and configure via envvars which log level suits best

Reply:
Gonna change it to use a logging package in another PR. Okay?

Reply:
But yeah, that's a good point!